### PR TITLE
Fix compatibility variable naming

### PR DESCRIPTION
**Pull request by hendrikmakait**
The abbreviation **gt** is commonly used for **greater than (>)**, whereas **greater than or equal (>=)** is commonly abbreviated as **ge**. Libraries following this naming convention include
* Python Standard Library's `operator` (https://docs.python.org/3/library/operator.html#operator.ge)
* `pandas` (https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.ge.html)
* `numpy` (https://numpy.org/doc/stable/reference/generated/numpy.ndarray.__ge__.html)

I suggest that we follow this naming convention.

- [x] Tests added / passed
- [x] Passes `pre-commit run --all-files`

### CODE REVIEW

File: dask/dataframe/_compat.py [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
-PANDAS_GT_202 = PANDAS_VERSION.release >= (2, 0, 2)
-PANDAS_GT_210 = PANDAS_VERSION.release >= (2, 1, 0)
+PANDAS_GE_131 = PANDAS_VERSION >= Version("1.3.1")
+PANDAS_GE_133 = PANDAS_VERSION >= Version("1.3.3")
+PANDAS_GE_140 = PANDAS_VERSION >= Version("1.4.0")
+PANDAS_GE_150 = PANDAS_VERSION >= Version("1.5.0")
+PANDAS_GE_200 = PANDAS_VERSION.major >= 2
+PANDAS_GE_201 = PANDAS_VERSION.release >= (2, 0, 1)
+PANDAS_GE_202 = PANDAS_VERSION.release >= (2, 0, 2)
+PANDAS_GE_210 = PANDAS_VERSION.release >= (2, 1, 0)
```
Comment (hendrikmakait) [inline dask/dataframe/_compat.py:19]: Renaming these variables is the main change, the remainder just follows suit.

File: dask/dataframe/rolling.py [hunk 2]
```diff
@@ -612,10 +612,10 @@ def apply(
         args = args or ()
         meta = self.obj._meta.rolling(0)
         if has_keyword(meta.apply, "engine"):
-            # PANDAS_GT_100
+            # PANDAS_GE_100
             compat_kwargs = dict(engine=engine, engine_kwargs=engine_kwargs)
```
Comment (crusaderky) [inline dask/dataframe/rolling.py:616]: This if is now always true; please simplify

File: dask/dataframe/tests/test_dataframe.py [hunk 3]
```diff
@@ -4255,7 +4255,7 @@ def test_idxmaxmin(idx, skipna):
 
     # https://github.com/pandas-dev/pandas/issues/43587
     check_dtype = not all(
-        (_compat.PANDAS_GT_133, skipna is False, isinstance(idx, pd.DatetimeIndex))
+        (_compat.PANDAS_GE_133, skipna is False, isinstance(idx, pd.DatetimeIndex))
```
Comment (crusaderky) [inline dask/dataframe/tests/test_dataframe.py:4258]:
```suggestion
        (PANDAS_GE_133, skipna is False, isinstance(idx, pd.DatetimeIndex))
```
plus import in header

**Review by crusaderky — CHANGES_REQUESTED** [review]

**Review by fjetter — APPROVED** [review]
I'm fine with the renaming without the additional stuff @crusaderky asked for. The renaming is not a functional change while all the other requested changes are changes to logic, even if it is about the removal of dead code.

Given the size of the diff alone, I would recommend to do the functional changes in a follow up PR. This way the changes are dedicated commits which makes things much easier in case one needs to bisect, blame, etc.

**Comment by hendrikmakait** [comment]
@crusaderky: I'll simplify the code in a follow-up as suggested by @fjetter.

**Pull request merged**